src/service/resources-impl.js

```diff
@@ -1054,6 +1073,10 @@ export class Resources {
    * @private
    */
   taskComplete_(task, success, opt_reason) {
+    if (!this.whenFirstResourceLaidOut_ && success) {
```

Comment:
Can you move it to the end of the method? Under the part that already checks success?